resource-managers/nomad/src/main/scala/org/apache/spark/scheduler/cluster/nomad/ExecutorTask.scala

```diff
@@ -81,6 +83,14 @@ private[spark] object ExecutorTask
     // The driver will lookup the actual log URLs
     task.addEnv("SPARK_LOG_URL_" + LOG_KEY_FOR_ALLOC_ID, "${NOMAD_ALLOC_ID}")
 
+    if (conf.getOption("spark.jars").nonEmpty) {
```

Comment:
I don't think you need to add a case for `spark.jars`.  I believe the value for this is used after job deployment and so it doesn't need to be handled in the "resource-manager"-specific code.

Reply:
I can remove that. 
But, wasn't the another expectation to set just "spark.jars", and the jars would be added both driver's and Executor's classpath ?

Reply:
yes but I believe the mechanism is different for each.  `spark.jars` are classloaded dynamically after the JVM is already running. The value of `extraClassPath` is passed as a JVM argument when the JVM is launched